### PR TITLE
Updated onOpen function in demo #11

### PR DESCRIPTION
**Pull request by DimaSavchenko**
Good way to import script that to use node_modules instead of download script by a link.
It will be nice if you commit build folder or merge my PR.
thanks

### CODE REVIEW

File: demo/scripts/openseadragon-imaginghelper.js [hunk 1]
```diff
@@ -693,8 +693,22 @@
...
     function onOpen() {
         this._haveImage = true;
-        this.imgWidth = this._viewer.viewport.contentSize.x;
-        this.imgHeight = this._viewer.viewport.contentSize.y;
+        var contentSizeViewport = {},
+          contentSize;
+        // use world if we can't use contentSize
+        if (!this._viewer.viewport.contentSize){
+            var homeBounds = this._viewer.world.getHomeBounds();
+            contentSizeViewport.x = homeBounds.width - homeBounds.x;
```
Comment (iangilman) [inline demo/scripts/openseadragon-imaginghelper.js]: I'm kind of new to this code; can you explain why you're subtracting the x from the width here?

**Comment by iangilman** [comment]
I see, you just updated the build in response to #12. Thank you!

**Comment by iangilman** [comment]
I've gone ahead and done a full build so we get all the changes and the min file:

https://github.com/msalsbery/OpenSeadragonImagingHelper/commit/66e6a5c48ea7efe24e567d54e5c35cfc6cd1cd51

**Comment by DimaSavchenko** [comment]
Thanks !

**Pull request merged**